src/viser/_gui_api.py

```diff
@@ -846,6 +851,85 @@ def add_gui_slider(
                 step=step,
                 initial_value=initial_value,
                 precision=_compute_precision_digits(step),
+                marks=[
+                    {"value": float(x[0]), "label": x[1]}
+                    if isinstance(x, tuple)
+                    else {"value": float(x)}
+                    for x in marks
+                ]
+                if marks is not None
+                else None,
+            ),
+            disabled=disabled,
+            visible=visible,
+            is_button=False,
+        )
+
+    def add_gui_multi_slider(
+        self,
+        label: str,
+        min: IntOrFloat,
+        max: IntOrFloat,
+        step: IntOrFloat,
+        initial_value: List[IntOrFloat],
+        disabled: bool = False,
+        visible: bool = True,
+        min_range: Optional[IntOrFloat] = None,
+        hint: Optional[str] = None,
+        order: Optional[float] = None,
+        fixed_endpoints: bool = False,
+        marks: Optional[
+            List[Union[Tuple[IntOrFloat, Optional[str]], IntOrFloat]]
+        ] = None,
+    ) -> GuiInputHandle[List[IntOrFloat]]:
```

Comment:
can we change the `list[T]` types to `tuple[T, ...]` where possible?

it doesn't seem like we need the ability to support mutation, right? like this would be invalid:
```python
handle = server.add_gui_multi_slider(...)
handle.append(5)  # add a new mark
```